### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by Morty67**

### CODE REVIEW

**Review by MikhailLyvak — APPROVED** [review]
Не знаю чи це потрібно, але хз чи треба тут перевірки, які ти добавив. Залишим на розсуд менторів)

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
-    pass
+def cinema_visit(
+    customers: list, hall_number: int, cleaner: str, movie: str
+) -> None:
+    clean_worker = Cleaner(cleaner)
+    cinema_hall = CinemaHall(hall_number)
+    customers_list = []
+    for customer in customers:
+        customer = Customer(customer["name"], customer["food"])
+        customers_list.append(customer)
```
Comment (wQuelS) [inline app/main.py:15]: this can be done in one line using list comprehension

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+        CinemaBar.sell_product(customer, customer.food)
```
Comment (wQuelS) [inline app/main.py:16]: make an instance of `CinemaBar` and use method not from class name

**Review by wQuelS — CHANGES_REQUESTED** [review]
don't change tests in any way!
I guess u've used `black` on the whole project. It is better to use it on the specific file you need.
Fix everything

**Pull request closed without merging**